core/org.osate.contribution.sei/src/org/osate/contribution/sei/names/SEI.java

```diff
@@ -84,4 +84,6 @@
 	public static final String PLATFORM = "platform";
 
 	public static final String PRICE = "price";
+
+	public static final String RESPONSE_TIME = "Response_Time";
```

Comment:
missing @ since tag

Reply:
Strange, I thought I had added this.  Fixed now.